space-framework/pkg/msclientlib/client.go

```diff
@@ -30,44 +29,45 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/clientcmd"
 
-	mcclientset "github.com/kubestellar/kubestellar/pkg/spaceclient/clientset"
 	spacev1alpha1 "github.com/kubestellar/kubestellar/space-framework/pkg/apis/space/v1alpha1"
-	ksclientset "github.com/kubestellar/kubestellar/space-framework/pkg/client/clientset/versioned"
+	mgtclientset "github.com/kubestellar/kubestellar/space-framework/pkg/client/clientset/versioned"
 	ksinformers "github.com/kubestellar/kubestellar/space-framework/pkg/client/informers/externalversions"
 )
 
 const defaultProviderNs = "spaceprovider-default"
 
 type KubestellarSpaceInterface interface {
 	// Space returns clientset for given space.
-	Space(name string, namespace ...string) (client mcclientset.Interface)
+
+	//Space(name string, namespace ...string) (client mcclientset.Interface)
+
 	// ConfigForSpace returns rest config for given space.
 	ConfigForSpace(name string, namespace ...string) (*rest.Config, error)
 }
 
 type multiSpaceClient struct {
 	ctx              context.Context
 	configs          map[string]*rest.Config
-	clientsets       map[string]*mcclientset.Clientset
-	managerClientset *ksclientset.Clientset
+	managerClientset *mgtclientset.Clientset
```

Comment:
BTW, in the previous line `configs map[string]*rest.Config`, it would be much better if there were a comment saying what strings appear in that map.

Reply:
Pushed comments to explain this hash